kolibri/plugins/management/assets/src/device_management/views/manage-content-page/channel-list-item.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+
+  <div class="channel-list-item">
+
+    <div class="thumbnail dtc">
+      <mat-svg
+        v-if="channel.thumbnail === ''"
+        category="action"
+        name="view_module"
```

Comment:
Should be using the [apps](https://material.io/icons/#ic_apps) material icon. It is slightly different. Mind also updating it in the Device navbar?

Also, would look nicer if you gave the square a `$core-grey` background color, similar to what is used in the cards under Learn. 

Note, you can also just use the `<content-icon/>` component instead of the `<mat-svg/>`.